phpBB/install/config.yml

```diff
@@ -0,0 +1,6 @@
+database:
```

Comment:
shouldn't be in this directory but in docs/ and with a different name.

May be a good idea to create a config_samples/ directory in docs btw